### ISSUE TITLE
kubestone controller manager crashloopbackoff

### ISSUE THREAD

**Issue by famoussou** [xridge/kubestone#183]
Step1- Deploy Kubestone to kubestone-system namespace with the following command:
$ kustomize build github.com/xridge/kubestone/config/default?ref=v0.5.0 | kubectl create -f -

**Results**
customresourcedefinition.apiextensions.k8s.io/drills.perf.kubestone.xridge.io created
customresourcedefinition.apiextensions.k8s.io/fios.perf.kubestone.xridge.io created
customresourcedefinition.apiextensions.k8s.io/iopings.perf.kubestone.xridge.io created
customresourcedefinition.apiextensions.k8s.io/iperf3s.perf.kubestone.xridge.io created
customresourcedefinition.apiextensions.k8s.io/pgbenches.perf.kubestone.xridge.io created
customresourcedefinition.apiextensions.k8s.io/qperves.perf.kubestone.xridge.io created
customresourcedefinition.apiextensions.k8s.io/sysbenches.perf.kubestone.xridge.io created

Step2- sudo kubectl describe pod -n kubestone-system

**Results**

Type     Reason   Age                     From                        Message
  ----     ------   ----                    ----                        -------
  Normal   Pulled   29m (x191 over 16h)     kubelet, VM Successfully pulled image "xridge/kubestone:latest"
  Warning  BackOff  3m58s (x4484 over 16h)  kubelet, VM Back-off restarting failed container

Step3- sudo kubectl  logs kubestone-controller-manager-xxxxxx-xxx -n kubestone-system

2020-07-10T08:10:14.708Z	INFO	controller-runtime.metrics	metrics server is starting to listen	{"addr": ":8080"}
2020-07-10T08:10:14.805Z	INFO	controller-runtime.controller	Starting EventSource	{"controller": "iperf3", "source": "kind source: /, Kind="}
2020-07-10T08:10:14.806Z	INFO	controller-runtime.controller	Starting EventSource	{"controller": "fio", "source": "kind source: /, Kind="}
2020-07-10T08:10:14.806Z	INFO	controller-runtime.controller	Starting EventSource	{"controller": "sysbench", "source": "kind source: /, Kind="}
2020-07-10T08:10:14.807Z	INFO	controller-runtime.controller	Starting EventSource	{"controller": "drill", "source": "kind source: /, Kind="}
2020-07-10T08:10:14.808Z	INFO	controller-runtime.controller	Starting EventSource	{"controller": "pgbench", "source": "kind source: /, Kind="}
2020-07-10T08:10:14.808Z	INFO	controller-runtime.controller	Starting EventSource	{"controller": "ioping", "source": "kind source: /, Kind="}
2020-07-10T08:10:14.809Z	INFO	controller-runtime.controller	Starting EventSource	{"controller": "qperf", "source": "kind source: /, Kind="}
2020-07-10T08:10:14.809Z	INFO	controller-runtime.controller	Starting EventSource	{"controller": "ycsbbench", "source": "kind source: /, Kind="}
2020-07-10T08:10:14.809Z	ERROR	controller-runtime.source	if kind is a CRD, it should be installed before calling Start	{"kind": "YcsbBench.perf.kubestone.xridge.io", "error": "no matches for kind \"YcsbBench\" in version \"perf.kubestone.xridge.io/v1alpha1\""}
github.com/go-logr/zapr.(*zapLogger).Error
******
2020-07-10T08:10:14.810Z	ERROR	setup	unable to create controller	{"controller": "YcsbBench", "error": "no matches for kind \"YcsbBench\" in version \"perf.kubestone.xridge.io/v1alpha1\""}
github.com/go-logr/zapr.(*zapLogger).Error
	/go/pkg/mod/github.com/go-logr/zapr@v0.1.0/zapr.go:128
main.main
	/workspace/main.go:146
runtime.main
	/usr/local/go/src/runtime/proc.go:200

**Comment by kulong0105** [comment]
i also met the same issue

**Comment by chrodriguez** [comment]
Same error here. I've managed to avoid it cloning the repo and installing using kustomize from local directory:

```
git clone https://github.com/xridge/kubestone
kustomize build kubestone/config/default | kubectl create -f -
```

**Comment by chrodriguez** [comment]
The problem is with revision v0.5.0. If command is executed from HEAD there is no error. I've didn't think when posted above solution: I was cloning HEAD and applying kustomize from HEAD instead of v0.5.0....

**Comment by kalinon** [comment]
correct. this should be fixed in #189
